Cybersecurity_Tools/ClipCrypt/tests/test_storage.py

```
@@ -0,0 +1,251 @@
+"""
+Tests for the storage module.
+"""
+
+import pytest
+import tempfile
+import shutil
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)